### PR TITLE
pacman4console 1.3 (new formula)

### PR DESCRIPTION
**Pull request by tomyun**
[pacman4console](https://sites.google.com/site/doctormike/pacman.html) is a Pac-Man clone running on the terminal.

- naming follows [Ubuntu](https://launchpad.net/ubuntu/+source/pacman4console) and [Debian](https://packages.debian.org/pacman4console).

### CODE REVIEW

File: pacman4console.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Pacman4console < Formula
+  desc "Pacman for console"
+  homepage "https://sites.google.com/site/doctormike/pacman.html"
+  url "https://sites.google.com/site/doctormike/pacman-1.3.tar.gz"
+  sha256 "9a5c4a96395ce4a3b26a9896343a2cdf488182da1b96374a13bf5d811679eb90"
+
+  def install
+    inreplace "pacman.h", "/pacman/", "/"
```
Comment (dunn) [inline pacman4console.rb]: Can we get an upstream ticket? 😀

Comment (mistydemeo) [inline pacman4console.rb]: Ping

Comment (tomyun) [inline pacman4console.rb]: No need to do `inreplace` actually. `Levels` are now installed in `#{pkgshare}/pacman`.

**Comment by tomyun** [comment]
Formula updated, but no `test` made up.

**Pull request closed without merging**